README.md

```diff
@@ -119,6 +119,28 @@ var lightbox1 = $('.lighbox-1 a').simpleLightbox();
 var lightbox2 = $('.lighbox-2 a').simpleLightbox();
 ```
 
+### Customization
+
+You can customize Simplelightbox by changing the style in simplelightbox.css.
+
+If you are using SASS, you can customize Simplelightbox with the following variables
+
+```SASS
+$sl-font-family: inherit;

```

Comment:
The ```font-family: Arial, Baskerville, monospace;``` was used to ensure correct rendering of the cross and arrow icons across browsers and operating systems. It should not be changed to ```inherit``` in my opinion. I would remove this variable entirely since the caption text already uses the global font family.